TestForSignUp/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿using OpenQA.Selenium;
```

Comment:
Does it really has to be a project for each test?

Reply:
Yes